esp-hal/src/uart.rs

```diff
@@ -1392,8 +1399,9 @@ where
     T: Instance,
     Dm: DriverMode,
 {
-    fn read(&mut self) -> nb::Result<u8, Self::Error> {
+    fn read(&mut self) -> embedded_hal_nb::nb::Result<u8, Self::Error> {
```

Comment:
So are we removing `nb` or are we not removing `nb`? :D You can just drop these impls.